source/index.html

```diff
@@ -2,11 +2,132 @@
   <body>
+  <!-- Если у нас есть блок в блоке, его название не обязательно начинать с названия блока-родителя?
+   Или не может быть в блоке других блоков, потому что они становятся его элементами?-->
+    <header class="page-header">
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item_active">
+            <a class="main-nav__link">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="photo.html" class="main-nav__link">Фото и видео</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="form.html" class="main-nav__link">Форма отзыва</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="" class="main-nav__link">HTML Academy</a>
+          </li>
+        </ul>
+        <button type="button" class="main-nav__button-close">Закрыть меню</button>
+      </nav>
+      <a class="header__logo">
+        <img src="https://via.placeholder.com/150x150" width="20" height="20" alt="Логотип сайта">
+      </a>
+    </header>
+    <main class="page-main">
+      <h1 class="visually-hidden">Добро пожаловать в Седону</h1>
+      <!-- Вставляю svg с разными шрифтами, а при стилизации с пом background-image
+       дорисовываю фотографию с горным ландшафтом? Логотип - это все надписи или только sedona?
+       На других стр сохраняется только это одно слово. Или это вообще не логотип, а просто
+       другая часть шапки, ведь в шапке сверху уже есть треугольный логотип?
+       И офтоп, но этот вопрос еще всплывет, забывала задать: чем неразрывный пробел лучше
+       переноса строки <br> (и лучше ли)? -->
+      <img src="https://via.placeholder.com/150x150" width="282" height="214" alt="Добро пожаловать в Седону">
+      <section class="advantages">
+        <h2 class="advantages__header">Седона &ndash; небольшой городок в Аризоне, заслуживающий большего!</h2>
+        <p class="advantages__text">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <!-- Разметила так, но можно же и просто как отдельные article или списком их сделать?
+             Список внутрь dl нельзя, только что увидела, исправлю -->
+        <dl class="advantages__list">
+          <div class="advantage__item">
+            <dt class="advantage__name">Настоящий городок</dt>
+            <p class="advantage__number">&ndash; 1 &ndash;</p>
+            <dd class="advantage__description">Седона не аттракцион для туристов, там течет своя жизнь</dd>
+            <!-- Имеет смысл к таким фото писать альтернативное описание? На
+               всякий случай написала, вроде в статье с первого курса такое было -->
+            <img src="" width="" height="" alt="Фотография двухэтажного мотеля с флагами на фоне гор">
+          </div>
+            <ul class="advantage__additional-list">
```

Comment:
Вот тут для вложенного списка можно дать второй класс, и его элементы стилизовать от второго класса:

```suggestion
            <ul class="advantages__additional-list additional-list">
```

И опять же, advantage**s**